### PR TITLE
Remove status flag from E2E

### PR DESCRIPTION
**Pull request by mkolesnik**
This flag made sense when everything was inside the e2e.sh script,
however since we migrated to shipyard which has distinct stages for each
operation, it doesn't make any sense to keep supporting this flag.

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To contribute to Submariner:
 
 1. Fork this repository
 2. Make your changes in the fork
-3. Test your changes by running `make ci e2e status=keep` which deploys [Submariner in a Kind 3 cluster setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md) and runs E2E tests
+3. Test your changes by running `make ci e2e` which deploys [Submariner in a Kind 3 cluster setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md) and runs E2E tests
```
Comment (mangelajo) [inline docs/contributing.md:28]:
```suggestion
3. Test your changes by running `make e2e status=keep` which deploys [Submariner in a Kind 3 cluster setup](https://github.com/submariner-io/submariner/blob/master/scripts/kind-e2e/README.md) and runs E2E tests
```

In this case, for developers, I'd instruct them to do the keep, so they can re-test again without re-deploying everything, may be we can add "5. Run `make cleanup` to tear down the test clusters once you're finished.

File: docs/testing.md [hunk 2]
```diff
@@ -177,7 +159,7 @@ After generating the Operator by running `make build-operator`, your newly gener
 is automatically fully integrated into the Submariner CI automation. Simply use
 the `deploytool` flag to the standard `make` commands.
 
-```make ci e2e status=keep deploytool=operator```
```
Comment (mangelajo) [inline docs/testing.md:180]: same here, it's for developers, I prefer we keep that by default which is more intuitive to continue with development..

File: docs/testing.md [hunk 3]
```diff
@@ -199,7 +181,7 @@ docker system prune --all
 #### Full example
 
 ```bash
-make ci e2e status=keep version=1.14.1 kubefed=true
+make e2e version=1.14.1 globalnet=true deploytool=operator
```
Comment (mangelajo) [inline docs/testing.md:184]: I'd keep here too.

File: docs/contributing.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ To contribute to Submariner:
...
 4. Submit a pull request (PR) to Submariner master branch
```
Comment (mangelajo) [inline docs/contributing.md:29]:
```suggestion
4. Submit a pull request (PR) to Submariner master branch
5. Run `make cleanup` to tear down the test clusters once you're finished.
```

File: docs/testing.md [hunk 5]
```diff
@@ -177,7 +159,7 @@ After generating the Operator by running `make build-operator`, your newly gener
 is automatically fully integrated into the Submariner CI automation. Simply use
 the `deploytool` flag to the standard `make` commands.
 
-```make ci e2e status=keep deploytool=operator```
+```make e2e deploytool=operator```
```
Comment (mangelajo) [inline docs/testing.md:162]:
```suggestion
```make e2e status=keep deploytool=operator```
```

File: docs/testing.md (hunk 3 above)
Comment (mangelajo) [inline docs/testing.md:184]:
```suggestion
make e2e status=keep version=1.14.1 globalnet=true deploytool=operator
```

File: scripts/kind-e2e/README.md [hunk 6]
```diff
@@ -24,42 +23,19 @@ Optional useful tools for troubleshooting:
...
-
-### 1. Ephemeral/Onetime
- 
-This mode will create e2e environment on a local workstation, run unit tests, E2E tests, and clean up the created resources afterwards.
-This mode is very convenient for CI purposes.
-    
+To run the tests simply execute the following:
 ```bash
-make ci e2e
+make e2e
```
Comment (mangelajo) [inline scripts/kind-e2e/README.md:28]:
```suggestion
make e2e status=keep
```
If we stop talking about the "ephemeral" option

File: scripts/kind-e2e/README.md [hunk 7]
```diff
@@ -115,7 +80,7 @@ docker system prune --all
 #### Full example
 
 ```bash
-make ci e2e status=keep version=1.14.1 kubefed=true
+make e2e version=1.14.1 globalnet=true
```
Comment (mangelajo) [inline scripts/kind-e2e/README.md:83]:
```suggestion
make e2e status=keep version=1.14.1 globalnet=true
```

**Comment by mangelajo** [comment]
If we go ahead with removal, I think we should at least document it somewhere.

File: docs/contributing.md (hunk 1 above)
Comment (mkolesnik) [reply docs/contributing.md:28]: What do you mean? The whole point is to remove `status=` flag.. There's also instructions on how to run cleanup..

**Pull request merged**